ansible-tests/validations/controller-ulimits-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: controller
+  vars:
+    metadata:
+      name: Check controller ulimits
+      description: >
+        This will check the ulimits of each controller.
+      groups:
+      - post-deployment
+    nofiles_default: 1025
```

Comment:
I think the default on many systems is 1024. Do we document somewhere than the max number of open file descriptors should be higher than 1024?